database/factories/InsegnamentoFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Model;
+use App\Models\CorsoDiStudi;
+use App\Models\Insegnamento;
+use Faker\Generator as Faker;
+
+$factory->define(Insegnamento::class, function (Faker $faker) {
+
+    $maxCorsoDiStudiProgressive = CorsoDiStudi::max('id');
+
+    return [
+        'codice_gomp'       => 1 + Insegnamento::max('codice_gomp'),
+        'nome'              => $faker->word,
+        'anno_accademico'   => '2017/2018',
```

Comment:
do we need to change this every year...?

Reply:
No, non cambia nulla di anno in anno (rispetto ai dati). Per semplicità ho focalizzato i test su un solo anno accademico, altrimenti avrei dovuto generare troppi dati.